docs/development/journalist_api.rst

```diff
@@ -112,6 +112,7 @@ Response 200 (application/json):
               "number_of_documents": 0,
               "number_of_messages": 2,
               "public_key": "-----BEGIN PGP PUBLIC KEY BLOCK-----\n\nmQINBFs2hGEBEACX9PSn9146bqup7MD3z4JLC2+m5GtXjOPmHVk7YRPwym7Q1XDx\n1exvXA1b17X6kj7TDPvBv8Gupro9BNAilPja+zB+m2JWKrdTjZYIWzZJ31WIC3Xm\nMs3V2dOZ1fCJlD+r2SiKLVzyODDpAoL42taxHXskhKgZvvUPsZv3abQctUOPtsWG\nKs9acPGGb/NnBVgPpdNzF7bKPpqqIHjMhb3WTEzGl8SYU/mfHx1DELzWmocB4v6s\nV4xMNKopyT44Or/ZeIGJf3SiTTsMSuU8IKfvzQKuCNT9IjWJmnYnYU+Zn/Zx/+q3\n1RWUs5z39e6OTT5qQwpxaharnJyM1u7vWY3R0rcZYkrAWQYgx/Ilf4/W1XSU7qmx\niH43mOupI1vQo0caJZwUvK83es2wmQsTNGJ1wqIU4pQU8nQzrlOuAWT3d/AjTXWh\nfFHMeRwfb2b2kRxp+hgFlC1hwpJG6o1+1kVUFUrh7N7Ln7WZi+UgQ23KGN1bU22D\nmY6fdEnssrODM8ly7AIHYhNOxtw/MnnWNlzt6n7gT26hN9VivXIczVxdkpV/vQz5\nng+olaLfXbf/yF/eTCmlsVdvALpDYYfO2VORcXe3JMTgXFzwQExz4auGdQlzH3ju\nmutOD5d0ETsgYP6lkO9wQrOqoqG/YnX+mUUc2H2wowYi5iFi11sLdbE7LQARAQAB\ntHxBdXRvZ2VuZXJhdGVkIEtleSA8NDRZR1oyUjc2NDNUWEJBNjVaS1JPNUQ2UUgy\nNlJKN05WREZNUUpWRlNNTTZXQTVXM1pEWE5VWUtHQlRFVVlHRkNBQkJVRURMUTdP\nS1M2NTdXS09HVUhGTFZETFE3NUdXVE9YNEQ0QT0+iQI/BBMBCgApBQJbNoRhAhsv\nBQkB4M5fBwsJCAcDAgEGFQgCCQoLBBYCAwECHgECF4AACgkQvPRLirPeGcnf/w//\nXOvsO/N6UtQasiE121xa0AwKtptaRUoprEUP8af3+tQ28Ibo+Io1LLEQDODS4Btu\n7rz2eXjhw6XjvtGYXjbOVtXVHqynPZu2eW+er5cbi+zlSjnN7RyLndsg5PZ457q0\n5b1p4olGEPVTFhjKmFoWcYGmfW2q/QvqD9uz4BQWpevMwpop0k7dWf6FI8h3LQk9\n6RWDP1lqgNFSvIQNZnsOv/uluuH+txMcvDGT2aDzpiPTkuXlmHQXo3GEjOq+bVcU\npbhREB+syJi9klM/ZqOixbDKGSOdZQjBg3n6Tc09K26Cczk/sAs85039L5QSZiEL\niERfSiMWhv3X18sh7z4NLuHV4U1V0sIRzBuyzNJB2bGo4OEudsQtgjceno84n8gz\nQojBqdrvlz1dzRsCQb8pHmc94UDyFKLU0oZAwoG9kkUWu60fmveLr56h7pojrw/9\nQeMdKg6nM6bSAQoI29zSEAuSzUUa6DpIlF0dDrlP/+NZVfOI7Fq2JVKPSmKnclpE\n1DsYw9ZrRJhYnm1O9wuO7unXPQtaWLql401VbUG9EXKoghnHtjPVzPyFgCPs2lPZ\n3uei1TPU0fkedvv+4m5cMg5+a0N1kZmuIABidFVWqdpTSaXY5U24BOuW1W5bYcgF\npx0IUtZOiYrKhbVZ+FA6Y2codyHnCSYqZ91cp2uvqj4=\n=K/aW\n-----END PGP PUBLIC KEY BLOCK-----\n",
+              "public_key_type": "PGP",
```

Comment:
Ah sorry I meant the `pub_key` should point to an object and not be two top level fields. This makes it easier for auto-deserializers in compiled languages (Rust/Scala come to mind) to be able to fully unmarshall the JSON response into a usable type. Without nesting it, macro/derived deserializers won't work. Even in Python land where it would come out as a dict, it's easier to pass the whole key to a function like:

```python
resp = client.get_source(source_id)
cyphertext = crypto.encrypt_to(resp.json['pub_key'], some_message)
```